apps/api/src/routers/saml.py

```diff
@@ -56,7 +57,7 @@ def _read_json(filename: str) -> dict[str, Any]:
     return OneLogin_Saml2_Settings(settings, custom_base_path=str(BASE_PATH))
 
 
-async def _prepare_saml_req(req: Request) -> dict[str, Any]:
+async def _prepare_saml_req(req: Request) -> dict[str, object]:
```

Comment:
I realize we did previously discuss this in HackAtUCI/HackUCI-Site#12.

Using `object` is safer than `Any` since pretty much anything can be done to/with something of type `Any` whereas the capabilities of `object` are limited. This specific return value is not currently used in our own code (only by library calls), so I decided it would be better to be safer with it than leave open the possibility of using it improperly later.

On this note, we could also have avoided using `Any` in `_get_saml_settings`, but that would require creating an interface for the SAML settings JSON files.